util/clientmetric/clientmetric.go

```diff
@@ -255,6 +255,10 @@ func EncodeLogTailMetricsDelta() string {
 	return enc.buf.String()
 }
 
+func ResetLastDeltaForTest() {
```

Comment:
Normally I'd say "all exported stuff needs proper docs" but given as how this is a ForTest func, meh.